src/views/pages/globalping/users.html

```diff
@@ -0,0 +1,448 @@
+								render ({ markers, position }) {
+									let totalCustomNumber = markers.reduce((sum, marker) => sum + (marker.markersCount || 0), 0);
+									let svg = createClusterMarkerSVG(totalCustomNumber);
+
+									let clusterDiv = document.createElement('div');
+									clusterDiv.setAttribute('data-probe-count', totalCustomNumber);
+									clusterDiv.setAttribute('data-zoom', map.zoom);
+
+									clusterDiv.style.position = 'relative';
+									clusterDiv.style.backgroundImage = `url('data:image/svg+xml;base64,${svg}')`;
+									clusterDiv.style.backgroundRepeat = 'no-repeat';
+									clusterDiv.style.backgroundPosition = 'center';
+									clusterDiv.style.backgroundSize = 'contain';
+
+									let { svgWidth, svgHeight } = getMarkerSize(totalCustomNumber, map.zoom);
+									clusterDiv.style.width = `${svgWidth}px`;
+									clusterDiv.style.height = `${svgHeight}px`;
+
+									return new google.maps.marker.AdvancedMarkerElement({
+										position,
+										content: clusterDiv,
+										zIndex: totalCustomNumber,
+									});
+								},
+							},
+							// eslint-disable-next-line no-undef
+							algorithm: new markerClusterer.SuperClusterAlgorithm({
+								radius: 40,
+							}),
+						});
+					}
+
+					if (!mq.matches) {
+						map.setCenter(MAP_CENTER_REG);
+						map.setZoom(MAP_ZOOM_REG);
+					} else {
+						map.setCenter(MAP_CENTER_ALT);
+						map.setZoom(MAP_ZOOM_ALT);
+					}
+				}
+			});
+
+
+			// close IWs on click on Map
+			google.maps.event.addListener(map, 'click', () => {
+				infoWindows.forEach(iw => iw.close());
+			});
+		},
+		createMapMarkerWithIW (key, markerData) {
+			let infoWindow = new google.maps.InfoWindow();
+			let svg = this.createMapMarkerSvg(markerData);
+			let [ lat, lng ] = key.split(', ');
+
+			// Collect InfoWindows
+			infoWindows.push(infoWindow);
+			infoWindow.setContent(this.createInfoWindowContent(markerData));
+
+			let markerContent = document.createElement('div');
+			markerContent.style.backgroundImage = `url('data:image/svg+xml;base64,${svg}')`;
+			markerContent.style.backgroundRepeat = 'no-repeat';
+			markerContent.style.backgroundPosition = 'center';
+			markerContent.style.backgroundSize = 'contain';
+			markerContent.setAttribute('data-probe-count', markerData.length);
+
+			let { svgWidth, svgHeight } = getMarkerSize(markerData.length, map.zoom);
+			markerContent.style.width = `${svgWidth}px`;
+			markerContent.style.height = `${svgHeight}px`;
+
+			// Create a new AdvancedMarkerView
+			let marker = new google.maps.marker.AdvancedMarkerElement({
+				map,
+				content: markerContent,
+				position: { lat: Number(lat), lng: Number(lng) },
+				zIndex: 1,
+			});
+
+			marker.markersCount = markerData.length || 0;
+
+			// Handle closing of all InfoWindows on a click on a marker
+			marker.addListener('click', () => {
+				infoWindows.forEach(iw => iw.close());
+				infoWindow.open({ map, anchor: marker });
+			});
+
+			return marker;
```

Comment:
Lower priority than the other comments, but if those functions are a 100% copy from the homepage (I assume they are), it might be a good idea to extract them to a separate file in `assets/js` which you just require and call here.

Reply:
This was copied from the network page. We can extract the map from network, home page and users page into a component. Needs a bit of cleanup and I suggest doing it in another PR if you agree. It being a component would mean we calculate user probes and filtered probes in case of network page and just pass it on. I assume extracting it to an asset file would mean exporting a bunch of functions to call separately. I think the component approach is better.